src/postprocessor.js

```diff
@@ -0,0 +1,15 @@
+import { escapeSelector } from '@unocss/core'
+
+const prefix = 'w-'
+const classify = (str) => `.${escapeSelector(str)}`
+
+export const postprocess = (shouldExternalize, _externalClasses = []) => {
+  const externalClasses = _externalClasses.map(classify)
+  return (obj) => {
+    if (shouldExternalize && externalClasses.includes(obj.selector)) return obj.entries = null
+    obj.entries.forEach(e => {
+      e[0] = e[0].replace(/^--un-/, `--${prefix}`);
```

Comment:
It looks like `--un-` comes from `colorResolver` util. Would it make sense to change that prefix to `w-` in that util?

Reply:
This function will catch all occurances of `--un`, so we don't need to worry about if it exists in some files (f.ex we have externalized most variants, so it could be happening there too!).

Reply:
Would it still make sense to update the prefix in our util? Just checking :)

Reply:
Doesn't matter either way I don't think?